src/reanimated2/commonTypes.ts

```diff
@@ -115,7 +115,11 @@ export interface NativeEvent<T> {
 export interface ComplexWorkletFunction<A extends any[], R>
   extends WorkletFunction {
   (...args: A): R;
-  __remoteFunction?: (...args: A) => R;
+  /**
+   * This prop is present only in DEV mode as a fallback for
+   * direct calls to RN thread functions on UI thread.
+   */
+  __functionInDEV?: (...args: A) => R;
```

Comment:
I don't like this name but I can't propose anything better at this moment.

Reply:
I've been through the desert on a horse with no name...

Reply:
Maybe I am not up to date with the current codebase, but according to my knowledge `__remoteFunction` is used in Release mode too 🤔 Are you sure about this change?

Reply:
You are right, I misinterpreted the comment and made changes in a rush. I'll fix it.

Reply:
Yes, but it's not passing the `category` argument, therefore this if
`else if (category === 'RemoteFunction') {` will never be entered